src/v2/styles/ImageCarousel/ImageCarousel-theme.scss

```diff
@@ -6,8 +6,21 @@
   --str-chat__image-carousel-stepper-color: var(--str-chat__text-color);
 }
 
+@media only screen and (max-device-width: 767px) {
```

Comment:
If someone knows a trick where I don't have to use 767px here (768-1) I'm happy to update this part

Reply:
True, thanks